src/project.rs

```diff
@@ -160,10 +160,23 @@ impl FuzzProject {
                                      -Cpasses=sancov \
                                      -Cllvm-args=-sanitizer-coverage-level=4 \
                                      -Cllvm-args=-sanitizer-coverage-trace-compares \
-                                     -Cllvm-args=-sanitizer-coverage-inline-8bit-counters \
-                                     -Cllvm-args=-sanitizer-coverage-pc-table \
                                      -Clink-dead-code"
             .to_owned();
+
+        match build.coverage_output_file {
+            Some(_) => {
+                rustflags.push_str(" -Zinstrument-coverage");
+            }
+            None => {
+                // These two flags currently cause a linking error when combined with `-Zinstrument-coverage`,
```

Comment:
Is there a bug that can be referenced here? It seems fairly unfortunate to have to tone down sanitization when the coverage is enabled. There is no obvious indication that one should not enable the `--coverage` option, for instance, and ideally there wouldn't be any need for it.

Reply:
I agree that removing those flags significantly affects the fuzzing, so I put them back. They do cause linking errors when fuzzing some larger programs. It's been difficult to minimize those examples into reportable test cases but I hope to find the time to try.